index.html

```diff
@@ -2694,12 +2711,196 @@ <h1>
         <h2>
 DID Resolution
         </h2>
+
+        <p>
+The <a>DID resolution</a> process takes a <a>DID</a> and resolution options as
+input, and produces a <a>DID document</a> and resolution metadata as output. The
+process utilizes the "Read" operation of the applicable <a>DID method</a>, as
+described in <a href="#read-verify"></a>. The <a>DID method</a>-specific details
+of how this process is accomplished is outside the scope of this specification.
+        </p>
+
+        <section>
+          <h3>
+Generic DID Resolution Algorithm
+          </h3>
+
+          <p>
+The generic DID resolution algorithm takes a <a>conforming DID</a>
```

Comment:
Same comment w.r.t the above, about the usage of "Generic"

Reply:
@tplooker See my reply above.